lib/logger_json/plug.ex

```diff
@@ -53,5 +70,13 @@ if Code.ensure_loaded?(Plug) do
         [val | _] -> val
       end
     end
+
+    defp extra_attributes(extra_attributes_fn, conn) do
+      if extra_attributes_fn do
+        extra_attributes_fn.(conn)
```

Comment:
I'm a bit worried here, if this function would ever be slow or crash it would crash logging so production system may be left without information what went wrong.

I guess there is a better way we can promote to solve your use case, instead a plug can be created that uses `Logger.set_metadata/1`. So if you have this plug in pipeline you can add any metadata you want without any changes to LoggerJSON codebase. Plus it's safe as if you crash in that plug logs would continue to work.

Reply:
Let me give that a try. But should solve the use-case in theory. Thanks!

Reply:
Thank you for submitting PR, I know you spent time on this so I'm here to help you to solve this use case. If something doesn't work feel free to ask :)